firmware/arduino/chessboard/movementFunctions.ino

```diff
@@ -150,15 +152,15 @@ bool moveAlongEdges(int startCol, int startRow, int endCol, int endRow)
   currRow = currPositionY / (stepsPerUnitSpace * 8);
 
   // Make initial move to first position. Move diagonally since it's faster.
-  if (!moveDirect(currCol, currRow, startCol, startRow))
+  if (!moveDirect(currCol, currRow, startCol, startRow, false))
```

Comment:
Please add comment explaining that since we're aligning to the start position, we want the electromagnet to be off.